### PR TITLE
Enable to change the calendar note font for the translation submod

### PR DESCRIPTION
**Pull request by proudust**
It makes it easy to change the calendar node font and wraps it if the note is long.

### CODE REVIEW

File: Monika After Story/game/zz_calendar.rpy [hunk 1]
```diff
@@ -84,6 +84,9 @@ init -1 python:
         # Color used for the note
         NOTE_COLOR = "#181818"
 
+        # Font used for the note
+        NOTE_FONT = "gui/font/m1.TTF"
```
Comment (Booplicate) [inline Monika After Story/game/zz_calendar.rpy:88]: We have our own font that's better, let's use it here.
```suggestion
        NOTE_FONT = "mod_assets/font/m1_fixed.ttf"
```

File: Monika After Story/game/zz_calendar.rpy [hunk 2]
```diff
@@ -687,6 +690,7 @@ M̼̤̱͇̤ ͈̰̬͈̭ͅw̩̜͇͈ͅa̲̩̭̩ͅs̙ ̣͔͓͚̰h̠̯̫̼͉e̗̗̮r
                                 size=note_text_size,
                                 color=note_color,
                                 outlines=[],
+                                xsize=110,
```
Comment (Booplicate) [inline Monika After Story/game/zz_calendar.rpy:693]: Notes must always fit in one line since multiline notes aren't supported. This is redundant at best.

**Review by Booplicate — CHANGES_REQUESTED** [review]
Please, rebase your PR to the content branch.

File: Monika After Story/game/zz_calendar.rpy (hunk 2 above)
Comment (proudust) [reply Monika After Story/game/zz_calendar.rpy:693]: I want to change this in a submod. It's easy to change if it's style. But is there any reason not to define it in style?

Comment (Booplicate) [reply Monika After Story/game/zz_calendar.rpy:693]: The only outcome of this change you'll get is this.
Capture
As you can see the notes overlap each other.

Also, you still need to [rebase](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-put-my-pull-request-in-the-wrong-branch-and-dont-want-to-suffer-the-wrath-of-devs-how-do-i-fix-it) the PR.

Comment (proudust) [reply Monika After Story/game/zz_calendar.rpy]: Ok, I fixed it.

**Review by Booplicate — APPROVED** [review]

**Review by ThePotatoGuy — APPROVED** [review]

**Pull request merged**